tests/kernel/mellanox_config.pm

```diff
@@ -42,7 +42,7 @@ sub run {
     # install dependencies
     zypper_call('--quiet in pciutils mstflint', timeout => 200);
 
-    my @devices = split(' ', script_output("lspci | grep -i infiniband.*mellanox |cut  -d ' ' -f 1"));
+    my @devices = split(' ', script_output("lspci | grep -i mellanox.*ConnectX-5 |cut  -d ' ' -f 1"));
```

Comment:
I would say, based on the previous experience during the review.  It will be better to catch whole output to lscpi and do regex in openQA.  If above fails, we dont know why as lspci output is hidden in pipes.